ahcore/entrypoints.py

```diff
@@ -204,8 +230,11 @@ def inference(config: DictConfig) -> None:
 
     # Convert relative ckpt path to absolute path if necessary
     checkpoint_path = config.get("ckpt_path")
-    if not checkpoint_path:
-        raise RuntimeError("No checkpoint inputted in config.ckpt_path")
+    jit_path = config.get("lit_module").model.get("jit_path", None)
```

Comment:
That's a neat recommendation. I also think this can move somewhere to utils.io since it's still related to loading weights and checking path to checkpoint.